generally recommend using remote bindings over `--remote` (#25256)
cloudflare/cloudflare-docs: src/content/docs/workflows/build/local-development.mdx

## Known Issues

> Workflows are not supported as [remote bindings](/workers/development-testing/#remote-bindings) nor with `npx wrangler dev --remote`.

Reviewer: ```suggestion
Workflows are not supported as [remote bindings](/workers/development-testing/#remote-bindings) or when using `npx wrangler dev --remote`.
```
